### DISCUSSION TITLE
部分音源无法匹配到

### DISCUSSION THREAD

**Discussion by memorydream** [1715173329/UnblockNeteaseMusic#423] [Q&A] [answered]
原文：
悔やむと書いてミライ 和 まふまふ的一些其他无版权歌曲，在旧的unm上可以匹配到音源，我确认过酷我上有这首歌的音源，但是新的unm找不到

原issue：
https://github.com/qier222/YesPlayMusic/issues/1149

**Comment by pan93412** [comment]
能給出 Netease 的音樂代號嗎⋯⋯我這裡測試正常，但它是抓 Netease 的音源。

**Comment by memorydream** [comment]
https://music.163.com/#/song?id=1807610094

**Comment by pan93412** [comment, marked as answer]
https://user-images.githubusercontent.com/28441561/147655270-7daa5372-5fb0-4ab1-9dbe-5ab477b3d0d4.mp4

我這裡測試正常。應該是 YPM 單純沒反應過來，按 F5 重新整理即可。採用的音源應該是 B 站的。

**Comment by pan93412** [comment]
對了，你有 Telegram 或是其他的聯絡方式嗎？我想把你拉進去 qier222 創的 YPM Telegram 群組中。 https://t.me/yesplaymusic

**Comment by memorydream** [comment]
抱歉，我没有telegram，我的discord: memorydream#8896
